triage/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "devDependencies": {
+        "jasmine": "~2.8.0"
+    }
+}
```

Comment:
nit: is there not supposed to be a trailing newline?

Reply:
my editor is automatically formatting this to not have a trailing newline for JSON blobs, I've added on in nano. :man_shrugging: